src/OpenLoco/src/Drawing/InvalidationGrid.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include <OpenLoco/Interop/Interop.hpp>
+#include <algorithm>
+#include <cstdint>
+
+namespace OpenLoco::Drawing
+{
+#pragma pack(push, 1)
+    struct ScreenInvalidationData
+    {
+        uint16_t blockWidth;
+        uint16_t blockHeight;
+        uint32_t columnCount;
+        uint32_t rowCount;
+        // TODO: Unused, remove when interop is no longer required.
+        int8_t columnShift;
```

Comment:
i assume you have checked that we have implemented and hooked all users of these 3?

Reply:
![ida_2023-08-25_12-12-57](https://github.com/OpenLoco/OpenLoco/assets/5415177/a8699ece-bfe9-4d0b-b6ab-1e31e32696b0)
They are only ever used in the set_dirty_blocks function but I'm doing it with ordinary multiplication, no need for the shifts.